lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -560,6 +560,10 @@
 
     .form-actions {
       @include padding($baseline 0 $baseline 0);
```

Comment:
NIT: no need to use the RTL ``include padding()`` mixin here (since all values are equal).